### PR TITLE
Added Polish translation

### PR DESCRIPTION
**Pull request by wojciechgabrys**
- Duplicated English translation (which looked like the most up-to-date)
- Translated all the strings (that's a lot of 🥔 trivia)
- Followed conventions and plugged the new language to the rest of the app
- Checked if everything was working as expected
- Ran `yarn run prettier:check` and `yarn run lint`

🇵🇱 _Smacznego!_

### CODE REVIEW

File: src/i18n/pl-PL/index.js [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+          label: 'Aplikacje, Mysz i Multimedia'
+        }
+      },
+      settingsPanel: {
+        title: 'Ustawienia konfiguratora',
+        fastInput: {
+          label: 'Szybkie wprowadzanie',
+          title: 'Ctrl + Alt + F',
+          help:
+            'Autmatyczne podświetlanie kolejnego klawisza po zdefiniowaniu funkcji popredniego.'
```
Comment (kbudaj) [inline src/i18n/pl-PL/index.js]:
```suggestion
            'Autmatyczne podświetlanie kolejnego klawisza po zdefiniowaniu funkcji poprzedniego.'
```

**Comment by wojciechgabrys** [comment]
@kbudaj Good catch, thanks! 😃

**Comment by yanfali** [comment]
@wojciechgabrys you good to go? I've had 3 positive reviews of your translation

**Comment by wojciechgabrys** [comment]
@yanfali Everything is good on my end, you can merge it in.

**Comment by wojciechgabrys** [comment]
@yanfali Everything is ok on my end, you can go ahead and merge it in :)

**Comment by yanfali** [comment]
Thanks for @kbudaj  hehetoja#8975 and bin#9281 for reviewing the translation

**Comment by Shehanka** [comment]
Seems like good PR

**Comment by ravijyoti3** [comment]
Initially made some typos but that fine now!

**Comment by raghavdhingra** [comment]
Looks good

**Pull request merged**